src/mc-fle2-payload-uev-private.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MONGOCRYPT_FLE2_UNINDEXED_ENCRYPTED_VALUE_PRIVATE_H
+#define MONGOCRYPT_FLE2_UNINDEXED_ENCRYPTED_VALUE_PRIVATE_H
+
+#include "mongocrypt-buffer-private.h"
+#include "mongocrypt-status-private.h"
+#include "mongocrypt-crypto-private.h"
+
+/**
+ * FLE2UnindexedEncryptedValue represents an FLE2 unindexed encrypted value.
+ * It is created client side.
+ */
+
+/* clang-format off */
+/*
+ * FLE2UnindexedEncryptedValue has the following data layout:
+ *   
+ * struct {
+ *   uint8_t fle_blob_subtype = 6;
+ *   uint8_t key_uuid[16];
+ *   uint8_t original_bson_type;
+ *   uint8_t ciphertext[ciphertext_length];
+ * } FLE2UnindexedEncryptedValue;
+ * 
+ * ciphertext is the output of: EncryptAEAD(key=K_Key, plaintext=ClientValue, associated_data=(fle_blob_subtype || key_uuid || original_bson_type))
```

Comment:
```suggestion
 * ciphertext is the output of:
 *   EncryptAEAD(
 *     key=K_Key,
 *     plaintext=ClientValue,
 *     associated_data=(fle_blob_subtype || key_uuid || original_bson_type))
```
Suggest remove clang-format disabling comments and reformat text as proposed or similar.